lua/projectlaunch/config.lua

```diff
@@ -64,4 +66,30 @@ function M.get_ecosystem_configs()
 	return cached_ecosystem_specific_configs
 end
 
+function M.reload_config()
+	cached_config = nil
+	cached_ecosystem_specific_configs = nil
+	M.get_project_config()
+	M.get_ecosystem_configs()
+end
+
+local function reload_config()
+	if options.get().auto_reload_config then
+		M.reload_config()
+	end
+end
+
+-- reload config when reload a saved session
+api.nvim_create_autocmd("SessionLoadPost", {
+	group = util.augroup,
+	callback = reload_config,
+})
+
+-- reload config when updated custom config
+api.nvim_create_autocmd("BufWritePost", {
+	group = util.augroup,
+	pattern = "*.json",
```

Comment:
Why did the pattern on this change?

Reply:
Because `options.get().config_path` is not working before options override. I just want to use custom `config_path`, but it always be default value `.projectlaunch.json`.

So I changed the pattern to match all json files, and reload config when  saving relatived buffers.